chipsec/utilcmd/tpm_cmd.py

```diff
@@ -88,15 +89,18 @@ def tpm_state(self):
         self._tpm.dump_intcap(self.locality)
         self._tpm.dump_intenable(self.locality)
 
-    def run(self):
+    def set_up(self):
         if self.func != self.tpm_parse:
             try:
                 self._tpm = tpm.TPM(self.cs)
             except TpmRuntimeError as msg:
                 self.logger.log(msg)
                 return
 
-        self.func()
-
+    def run(self):
+        try:
+            self.func()
+        except Exception:
+            return ExitCode.ERROR
```

Comment:
Instead of returning ExitCode.Error, set self.ExitCode = ExitCode.Error. This should fix the "Explicit returns mixed with implicit returns" error.
```suggestion
            self.ExitCode = ExitCode.ERROR
```